packages/embark/src/lib/modules/ipfs/index.js

```diff
@@ -20,29 +20,31 @@ class IPFS {
     this.webServerConfig = embark.config.webServerConfig;
     this.blockchainConfig = embark.config.blockchainConfig;
 
-    if (this.isIpfsStorageEnabledInTheConfig()) {
-      this.setServiceCheck();
-      this.registerUploadCommand();
-
-      this.events.request("processes:register", "ipfs", (cb) => {
-        this.startProcess(() => {
-          this.addStorageProviderToEmbarkJS();
-          this.addObjectToConsole();
-          this.events.emit("ipfs:process:started");
-          cb();
-        });
-      });
+    if (!this.isIpfsStorageEnabledInTheConfig()) {
+      return this.events.emit("ipfs:process:started", false);
```

Comment:
The consequences of emitting `false` aren't immediately clear to me, as opposed to just `emit("ipfs:process:started")` done elsewhere. Can you explain?

Reply:
It basically does nothing in our case, but if one of the listeners wanted, it could get the state with `.on('ipfs:process:started', (state) => {});`  then do something if `false`. 

TLDR: added the `false` just in case and also because it looked weird to say "I'm ready" when it's actually not enabled